### PR TITLE
fix pfTableView: border style not in line with PatternFly

### LINKED ISSUE #663: pfTableView: border style not in line with PatternFly
The table borders are thicker than they should be.

image

### PR DESCRIPTION
**Pull request by akieling**
fix #663 

Make the pfTableView borders thinner according to PatternFly's [Table View](http://www.patternfly.org/pattern-library/content-views/table-view/).

image

@cshinn

### CODE REVIEW

File: src/pagination/pagination.less [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
-  align-items: center;
-  .per-page-label {
-    padding-left: 10px;
+.content-view-pf-pagination {
+  border-top: none;
+  .form-group {
+    align-items: center;
+    .per-page-label {
+      padding-left: 10px;
+    }
```
Comment (jeff-phillips-18) [inline src/pagination/pagination.less]: Shouldn't this be in Patternfly styling? Not sure why the original was here. @dtaylor113 ?

**Comment by jeff-phillips-18** [comment]
Commit message needs to be formatted for semantic release. See https://github.com/patternfly/angular-patternfly#git-commit-guidelines

**Comment by akieling** [comment]
Formatted the commit message according to semantic releases

File: src/pagination/pagination.less (hunk 1 above)
Comment (dtaylor113) [reply src/pagination/pagination.less]: Without the less, I get:

image

I'd have to dive deeper into the css used in the a-pf pagination component.  I'm fairly certain it is very close to the css used in pf core.

**Comment by dtaylor113** [comment]
@jeff-phillips-18, are you ready to accept this PR?  Filed a issue for the less comment.  -thanks

File: src/pagination/pagination.less (hunk 1 above)
Comment (akieling) [reply src/pagination/pagination.less]: Can this change be done in a separate PR so we can merge & close this one?

Comment (jeff-phillips-18) [reply src/pagination/pagination.less]: Looks like the real issue here is that the pagination component should have the 'table-view-pf-pagination' selector as well as the 'content-view-pf-pagination'. If we add that for the pagination component when used for a table this change would not be necessary.

**Comment by akieling** [comment]
Your requested changes have been done.

**Comment by dtaylor113** [comment]
@jeff-phillips-18, need your approval -thanks

**Pull request merged**